crypto/store/loader_file.c

```diff
@@ -748,7 +755,7 @@ static OSSL_STORE_LOADER_CTX *file_open(const OSSL_STORE_LOADER *loader,
         }
     } else {
         BIO *buff = NULL;
-        char peekbuf[4096];
+        char peekbuf[64];
```

Comment:
If we really do need to peek a full 4kbytes into the file to decide if it has PEM content, we should probably put the buffer on the heap.